dhall-json/src/Dhall/JSONToDhall.hs

```diff
@@ -0,0 +1,392 @@
+{-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE RecordWildCards     #-}
+{-# LANGUAGE PatternGuards       #-}
+{-# LANGUAGE FlexibleContexts    #-}
+{-# LANGUAGE FlexibleInstances   #-}
+{-# LANGUAGE LambdaCase          #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Dhall.JSONToDhall where
```

Comment:
Since this module is publicly visible it's probably worth adding a module header to document what it is used for and also an explicit export list so that we can safely change internal utility functions in this module without breaking the public API

Reply:
Good point. I've added the explicit exports and moved the docs over to the public module.